.github/workflows/wheel_manylinux.yml

```diff
@@ -0,0 +1,71 @@
+name: wheel - manylinux2014
+
+on:
+  push:
+    tags:
+      - '*'
+jobs:
+  wheel_manylinux:
+    name: manylinux2014
+    runs-on: ${{ matrix.os }}
+    container:
+      image: quay.io/pypa/manylinux2014_x86_64:latest
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        python-version: [cp37-cp37m, cp38-cp38]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        run: |
+          echo "::add-path::/opt/python/${{ matrix.python-version }}/bin/"
+      - name: check C++ compiler and Python
+        run: |
+          gcc --version
+          python --version
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade --no-cache-dir pip
+          python -m pip install --no-cache-dir "poetry<2,>=1.0"
+          python -m pip install --no-cache-dir "mypy-protobuf~=1.0"
+      - name: Build libraries
+        run: |
+          make
+      - name: Build wheel
+        run: |
+          python -m poetry build -v
+      - name: Fix wheel
+        env:
+          PLAT: manylinux2014_x86_64
+        run: |
+          for whl in dist/*.whl; do
+            auditwheel repair "${whl}" --plat "${PLAT}" -w "./dist"
```

Comment:
Looking at the multibuild and auditwheel code, I don't see them using ``--plat``. Perhaps they autodetect the platform (which would make sense).

Reply:
I think that plat is somewhere. I can try without it. Maybe the docker has already that as a default?

Reply:
I'm happy to change things from our current setup (see httpstan-wheels) if there's a good reason to do so. Otherwise I'd prefer to keep things the same. I barely understand all the details here so I think it's best to stick to what we know works (i.e., httpstan-wheels)

Reply:
I was thinking this was a minor issue but now I'm not so sure. I think we should be explicit here and upload the platform-specific wheel by name. We know what the filenames look like. For linux the filename looks like this: `httpstan-*-manylinux*_x86_64`. For macOS it looks like this: `httpstan-*-macosx_*_intel.whl`.

The reason for this is the following -- we want to avoid accidentally uploading the un-repaired "any" wheel. This mistake just happened in httpstan-wheels with 2.0.0. You can see on PyPI that a "any" whl was uploaded. It probably has linux specific shared libraries in it. So it's broken on macOS.